woocommerce/woocommerce
# Add 'placeholder' attribute to taxonomy block and add 'placeholder' a…

…ttribute to category field in Simple Product Template

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





I noticed that the placeholder for categories was missing. This was probably forgotten while implementing the taxonomy block, since it is referenced in the wireframes.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Enable the new product editor at /wp-admin/admin.php?page=wc-settings&tab=advanced&section=features
2. Go to Products > Add
3. Go to Organization tab
4. Delete the "Uncategorized" category
5. Check that you see the placeholder



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
